### ISSUE TITLE
[css-inline-3] Canvas 2D's textBaseline = "middle" neccessitates an em-middle baseline?

### ISSUE THREAD

**Issue by annevk** [w3c/csswg-drafts#5431]
It's described as "The middle of the em square". 

This is a follow-up to #5312 that only looked at the `measureText()` API which doesn't expose this.

cc @yiyix @kojiishi @macnmm @litherum

**Comment by annevk** [comment]
Maybe an alternative is that I describe it as the middle of the em-over and em-under baselines, but we might as well have a term for it?

**Comment by fantasai** [comment]
@annevk I think you should be able to use the `central` baseline definition for this, no? Note that `vertical-align: middle` is defined as using the "ex-middle" baseline, so there will be some inconsistency in terminology between Canvas and CSS.

**Comment by annevk** [comment]
What is ex-middle? I don't see it in https://drafts.csswg.org/css-inline/#baseline-types.

And are we sure the middle of em-over and em-under is equal to the middle of ideographic-under and ideographic-over? If so, that's probably worth noting in the specification, but it seems unlikely that would be the case?

**Comment by annevk** [comment]
Yeah, though it seems a bit weird that we would sometimes give that an explicit baseline definition (e.g., as for central) and sometimes not.

**Comment by fantasai** [comment]
@annevk Yeah, but it's a simple relationship between two existing terms (whereas defining what each of these terms are is not). I'll note CSS doesn't use these at all so defining any of them here is a bit weird to begin with :)

**Comment by annevk** [comment]
We ended up defining this as:

> Halfway between the em-over baseline and the em-under baseline

**Comment by macnmm** [comment]
halfway between em-over and -under works for me.
